socorro/external/es/super_search_fields.py

```diff
@@ -379,6 +379,20 @@ def get(self):
         "query_type": "enum",
         "storage_mapping": None,
     },
+    "application_build_id": {
+        "data_validation_type": "int",
+        "description": "Product application's build ID.",
+        "form_field_choices": [],
+        "has_full_version": False,
+        "in_database_name": "application_build_id",
+        "is_exposed": True,
+        "is_returned": True,
+        "name": "application_build_id",
+        "namespace": "processed_crash",
+        "permissions_needed": [],
+        "query_type": "number",
+        "storage_mapping": {"type": "long"},
```

Comment:
This is soooooo much "string" boilerplate. Some day it'd be great to fix that.

This is structured the same as `build_id`. I figured users are likely to want to do the same things with it.